manim/mobject/geometry.py

```diff
@@ -339,6 +339,11 @@ def move_arc_center_to(self, point):
     def stop_angle(self):
         return angle_of_vector(self.points[-1] - self.get_arc_center()) % TAU
 
+    def set_radius(self, new_radius: float):
+        self.radius = new_radius
+        self.scale(self.radius)
```

Comment:
I think later it should not be `self.scale(self.radius)` but something like `self.set_height = (self.radius)` as otherwise it would add multiple times. I just chose `scale` now for debugging.